Header header
sensor_msgs/PointCloud2 point_cloud
foveated_stereo_ros/Covariance[] covariances
